Tests.Integration.Presentation.Web/ItSystem/V2/ItSystemLocalOptionTypesInternalV2ApiTest.cs

```diff
@@ -0,0 +1,136 @@
+﻿
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.ItSystem;
+using Core.DomainModel.Organization;
+using Newtonsoft.Json;
+using Presentation.Web.Models.API.V2.Internal.Request.Options;
+using Presentation.Web.Models.API.V2.Internal.Response;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Integration.Presentation.Web.Tools.Extensions;
+using Tests.Integration.Presentation.Web.Tools.Internal.ItSystem;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.ItSystem.V2
+{
+    public class ItSystemLocalOptionTypesInternalV2ApiTest: WithAutoFixture
+    {
+        private const int CvrLengthLimit = 10;
+        private const string BusinessTypesUrlSuffix = "business-types";
+
+        [Fact]
+        public async Task Can_Get_Local_Business_Types()
+        {
+            var organization = await OrganizationHelper.CreateOrganizationAsync(A<int>(), A<string>(), A<string>().Truncate(CvrLengthLimit), OrganizationTypeKeys.Kommune, AccessModifier.Public);
```

Comment:
Can extract this call into a private helper method, so you don't have to provide the autofixture arguments each time